### PR TITLE
feat(Business): Add crud

### PR DESCRIPTION
**Pull request by Gio85**
Add crud

### CODE REVIEW

File: src/api/client/index.ts [hunk 1]
```diff
@@ -1,14 +1,17 @@
+require('dotenv').config()
```
Comment (useless-stuff) [inline src/api/client/index.ts:1]: don't hate me
did you try with
```
import {config} from 'dotenv' 
config()
```

Comment (useless-stuff) [reply src/api/client/index.ts:1]: or 
```
import dotenv from 'dotenv'
dotenv.config()
```

File: src/api/client/endpoints/business.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    return this.client.post<IBusiness>(this.path, payload)
+  }
+
+  /**
+   * This method will return the business model
+   * @param businessId
+   */
+  public async get(businessId?: string): Promise<IApiResponse<IBusiness>> {
+    if (businessId) {
+      return this.client.get<IBusiness>(`${this.path}/${businessId}`)
```
Comment (Gio85) [inline src/api/client/endpoints/business.ts:22]: keep this and make id required

**Pull request merged**